daemon/src/maker.rs

```diff
@@ -353,7 +354,7 @@ where
                 .notify_interval(Duration::from_secs(5), || oracle::Sync)
                 .unwrap(),
         );
-        let fan_out_actor = fan_out::Actor::new(&[&cfd_actor_addr, &cfd_actor_addr])
+        let fan_out_actor = fan_out::Actor::new(&[&cfd_actor_addr, &monitor_addr])
```

Comment:
❓ I have not noticed this in the code base, what does this line do?

Reply:
From the docs:

> /// A fan-out actor takes every incoming message and forwards it to a set of other actors.

We are using this pattern with the oracle actor to forward the attestation to both, the cfd actor and the monitoring actor without the oracle actor having to loop over a list of addresses.